paws.common/NEWS.md

```diff
@@ -2,6 +2,7 @@
 
 * Fix Content-Md5 being modified by user
 * Add logging system to help with debugging paws issues
+* Add s3 redirects from error message
```

Comment:
I think this would be better phrased as "Automatically redirect S3 requests when requests are initially made to the wrong region"